pkgs/development/python-modules/phonemizer/default.nix

```diff
@@ -31,6 +32,12 @@ buildPythonPackage rec {
       libespeak = "${lib.getLib espeak-ng}/lib/libespeak-ng${stdenv.hostPlatform.extensions.sharedLibrary}";
       # FIXME package festival
     })
+    # This patch is needed for python3Packages.misaki. See https://github.com/thewh1teagle/espeakng-loader?tab=readme-ov-file#usage-with-phonemizer.
```

Comment:
Why don't we use phonemizer-fork, as they suggest?
Also, might be worth linking to https://github.com/bootphon/phonemizer/pull/191.

Reply:
phonemizer-fork is not currently packaged in nixpkgs and I figured that it would be preferable to avoid the duplication and just patch the existing library instead.

Agreed on linking, I've added that in the comment.